fluss-client/src/test/java/com/alibaba/fluss/client/admin/ClientToServerITCaseBase.java

```diff
@@ -110,8 +110,6 @@ private static Configuration initConfig() {
         conf.setInt(ConfigOptions.DEFAULT_REPLICATION_FACTOR, 3);
         // set a shorter interval for testing purpose
         conf.set(ConfigOptions.KV_SNAPSHOT_INTERVAL, Duration.ofSeconds(1));
-        // set a shorter max lag time to make tests in FlussFailServerTableITCase faster
-        conf.set(ConfigOptions.LOG_REPLICA_MAX_LAG_TIME, Duration.ofSeconds(10));
```

Comment:
Now, we don't need to set to a shorter max lag time since offline replica will be removed from isr by coordinator after it knows the replica is offline.